ci/README.md

```diff
@@ -1,29 +1,8 @@
-# Continuous integration / analysis directory
+# Continuous integration tools
 
-[![Build Status](https://travis-ci.com/manubot/rootstock.svg?branch=master)](https://travis-ci.com/manubot/rootstock)
+This directory contains tools and files for continuous integration (CI).
+Specifically, [`deploy.sh`](deploy.sh) runs on successful `master` branch builds that are not pull requests.
+The contents of `/webpage` are committed to the `gh-pages` branch.
```

Comment:
Should these paths be relative to this directory?  `../webpage` and `../output` would also make it clear these are relative paths, similar to the earlier comment about the `/ci` directory.

Reply:
Done in https://github.com/manubot/rootstock/pull/243/commits/ae04618a19933ef410d18bef4e229416e1b64022